src/sdk/models/lineItem.ts

```diff
@@ -0,0 +1,35 @@
+import { given } from "@nivinjoseph/n-defensive";
+
+
+
+export class LineItem 
+{
+    private static _count = 200;
+    private readonly _itemId: string;
+    private readonly _productName: string;
+    private readonly _quantity: number;
+    private readonly _mrp: number;
+
```

Comment:
It is better to have 2 line spacing here and in line 20 and remove the spacing at 34

Reply:
okay